recipes/jsonrpcclient/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "jsonrpcclient" %}
+{% set version = "3.3.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6f2c5d6bed0d37e4f1f7e76c6e0ae2669ef2facd709508398cc54086b83a6dcf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >3
+    - pip
+  run:
+    - python >3
+
+test:
+  imports:
+    - jsonrpcclient
+
+about:
+  home: https://github.com/bcb/jsonrpcclient
+  license: MIT
+  license_family: MIT
+  license_file: https://github.com/bcb/jsonrpcclient/blob/master/LICENSE
```

Comment:
https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually